src/Compute/Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -100,6 +100,20 @@ public override void ExecuteCmdlet()
                                 checkFlexibleOrchestrationModeParamsDefaultParamSet(parameters);
                             }
 
+                            if (parameters.VirtualMachineProfile.SecurityProfile.SecurityType == "TrustedLaunch" || parameters.VirtualMachineProfile.SecurityProfile.SecurityType =="ConfidentialVM")
```

Comment:
there are no null check protections on this line. The fields in 'parameters' should have a ? at the end of them except SecurityType, that should work.

Reply:
Same for below if checks

Reply:
Done added ? at end

Reply:
No null checks needed